src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -137,8 +138,22 @@ export class HuiCardPicker extends LitElement {
       return;
     }
 
-    this._unusedEntities = computeUnusedEntities(this.hass, this.lovelace);
-    this._usedEntities = [...computeUsedEntities(this.lovelace)];
+    const usedEntities = computeUsedEntities(this.lovelace);
+    const unusedEntities = calcUnusedEntities(this.hass, usedEntities);
+
+    this._usedEntities = [...usedEntities].filter(
```

Comment:
`filter` already creates a new array. No need to copy it.

Reply:
I'm confused by what you mean by copy it

Reply:
`[...usedEntities]` is creating a copy? 

Oh, I guess it's a set here ?

Reply:
Yea. and I think we should convert this here so that we can filter them before sending to get config